src/planscape/eval/compute_conditions_test.py

```diff
@@ -246,20 +246,23 @@ def test_score_condition_known_element_returns_value(self):
         self.assertIsNotNone(score)
         self.assertIsNotNone(element)
         if element is not None:
-           expected = cc.score_element(FakeConditionReader(), element, ConditionScoreType.IMPACT, recompute=True)
-           self.assertTrue(np.all(score == expected))
+            expected = cc.score_element(FakeConditionReader(
+            ), element, ConditionScoreType.IMPACT, recompute=True)
```

Comment:
Formatting looks strange here...reformat?

Reply:
Odd, that's what the formatter suggests. Manually fixed.